test/test_jit.py

```diff
@@ -8181,6 +8181,26 @@ def test_indexing_out_of_bounds_neg():
         self.checkScriptRaisesRegex(test_indexing_out_of_bounds_pos, (), Exception,
                                     "out of range")
 
+    def test_namedtuple_attr(self):
+        def f(x):
+            return x.max(dim=1).indices + torch.max(x, dim=1).indices
+
+        sf = torch.jit.script(f)
+        self.assertExpectedGraph(sf.graph)
```

Comment:
We're trying to limit the number of expect files we have since they've led to a number of issues, the `checkScript` call below should be enough so you could just delete this line